scripts/travis-build-test.sh

```diff
@@ -1,10 +1,17 @@
 #!/usr/bin/env bash
 
 if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-  export {no_proxy,NO_PROXY}="127.0.0.1,localhost"
-	export DISPLAY=:99.0
-  sh -e /etc/init.d/xvfb start
-  sleep 3
+    export {no_proxy,NO_PROXY}="127.0.0.1,localhost"
+    export DISPLAY=:99.0
+    sh -e /etc/init.d/xvfb start
+    sleep 3
+
+    echo 'Travis Screen Resolution:'
+    xdpyinfo | grep dimensions
 fi
 
 npm run test
+
+if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
```

Comment:
@SimplyAhmazing should we include `Windows` as well?

Reply:
Tests are run on Windows via [appveyor.yml](https://github.com/SimplyAhmazing/zulip-electron/blob/setup-e2e-tests/appveyor.yml#L24). So the tests run on Windows & Mac but not Linux. The test hung on Linux for me, it may be that Travis Linux has cached an older version of electron, just your personal setup. But I haven't had a chance to investigate this. So for now only Windows & Mac.

Reply:
@akashnimare You can see the test run passing on Appveyor for my clone of the project [here](https://ci.appveyor.com/project/SimplyAhmazing/zulip-electron/build/build75)